lib/views/text.js

```diff
@@ -66,12 +67,15 @@ var TextView = View.extend({
         }
     },
 
-    format_values: function(values) {
-        var val2 = _.map(values, function(value) {
-            if (_.isDate(value)) {
-                value = value.toISOString();
+    format_values: function(values_) {
+        var val2 = _.map(values_, function(value) {
+            if (typeof(value) === 'undefined') {
+                return '';
+            } else if (values.isArray(value) || values.isObject(value)) {
+                return values.inspect(value);
```

Comment:
I’m not sure about this bit, we should rather make `values.toString` work in such a way that this isn’t necessary.

I started a [discussion on Gitter](https://gitter.im/juttle/juttle?at=569783b7a03e28ad1adea550).